rest-api-json-bulk/build.gradle.kts

```diff
@@ -6,12 +6,14 @@ plugins {
 
 val ktor_version : String by project
 val json_bulk_access_version: String by project
-val api_gen_version: String by project
+val modelix_version: String by project
+val modelix_metamodel_version: String by project
 
 val openApiFile = layout.projectDirectory.file("../openapi/openapi.yaml")
 
 dependencies {
-    implementation(project(":mps:solutions:University.Schedule.api"))
+    implementation(project(":mps:metamodel"))
+    implementation("org.modelix:metamodel-runtime:$modelix_metamodel_version")
```

Comment:
Shouldn't that be a dependency of the metamodel subproject already? The code there probably isn't usable without the runtime?

Reply:
yes, we can drop line 16 here

Reply:
Should be solved with using `api` instead of `implementation` in the metamodel module, right?

Reply:
ah yes, that makes sense!